bucket/stevedore-np.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.8.0",
+    "description": "Docker distribution for Windows that Just Works",
```

Comment:
```suggestion
    "description": "Docker distribution for Windows that aims to provide a frictionless Docker experience on Windows",
```

Reply:
That ends up making it kinda repeat itself

Reply:
emm... How about "Docker distribution ~for Windows~ that aims to provide a frictionless Docker experience on Windows". I just thought that `that Just Works` seems vague.

Reply:
Yeah that sound's better.

Reply:
I originally just copied it from the repo description. Is it okay to change it? @slonopotamus

Reply:
Sure